app/controllers/ems_common.rb

```diff
@@ -185,6 +185,18 @@ def show
                       pluralize(@view.extras[:total_count] - @view.extras[:auth_count], "other #{title.singularize}") +
                       " on this " + ui_lookup(:tables => @table_name)
       end
+    elsif @display == "container_builds" || session[:display] == "container_builds" && params[:display].nil?
+      title = ui_lookup(:tables => "container_builds")
+      drop_breadcrumb(:name => @ems.name + " (All #{title})",
+                      :url  => show_link(@ems, :display => @display))
+      @view, @pages = get_view(ContainerBuild, :parent => @ems)  # Get the records (into a view) and the paginator
+      @showtype = @display
+      if @view.extras[:total_count] > @view.extras[:auth_count] && @view.extras[:total_count] &&
```

Comment:
@DNNX I changed the order. I think that initially I copied it from the section above for container_images (which have it the wrong way as you stated above, I guess maybe it never came up because those variables can't be nil). It seems to me like the whole section from line 70 to 270 needs be refactored as it is just a copy of the first part repeated 20 times, It is for a different patch though.